canonical-web-and-design/vanilla-framework
# Remove .npmignore

## Done

Removes .npmignore in favour of defining list of files to package in package.json

Fixes #2686

## QA

- Pull code
- Run `npm publish --dry-run`
- Check if only scss files are included in the package
